packages/host/app/lib/current-run.ts

```diff
@@ -454,7 +454,7 @@ export class CurrentRun {
         error = { type: 'error', error: typesMaybeError.error };
       } else {
         let err = new Error(`bug: should never get here`);
-        deferred.reject(err);
+        deferred.fulfill();
```

Comment:
if we don't think we'll ever get here then shouldn't we reject with the `err` so we know if we do end up in this place?

Reply:
> if we don't think we'll ever get here then shouldn't we reject with the `err` so we know if we do end up in this place?

We made `reject` private inside the `Deferred` class, and it seemed to me that maybe `fulfill` was also handling the error case. If that's not correct, how to handle this? Shall I add a public `reject` in Deferred?

Reply:
Oh my bad! I didn’t realize fulfill handled both. You should leave it as is.